rpc/wrtc_signaling_answerer.go

```diff
@@ -86,14 +92,45 @@ const (
 )
 
 // Start connects to the signaling service and listens forever until instructed to stop
-// via Stop.
+// via Stop. Start cannot be called once more than once before a Stop().
 func (ans *webrtcSignalingAnswerer) Start() {
 	ans.startStopMu.Lock()
 	defer ans.startStopMu.Unlock()
 
-	for i := 0; i < defaultMaxAnswerers; i++ {
-		ans.startAnswerer()
-	}
+	ans.answerMu.RLock()
+	ans.answerWorkers.Add(1)
```

Comment:
[q] Are you now using the `answerWorkers` wait group to keep track of the connection establishment goroutine + the eventual, two answer workers? That seems fine to me, but wanted to check.

Reply:
yep, updated some var names as well to reflect